### PR TITLE
Add a note about full/trimmed ruby version in legacy file

### PR DESCRIPTION
**Pull request by azhi**
Related to https://github.com/asdf-vm/asdf/issues/352.
This PR only documents current behaviour.

### CODE REVIEW

**Comment by Stratus3D** [comment]
Do any version managers support the `.ruby-version` without a complete version? It doesn't look like rbenv supports it.

**Comment by azhi** [comment]
Yep, rvm does. It is mentioned in install docs https://rvm.io/rvm/basics#installing-rubies, under NOTE:

> If you use just the Major.Minor version numbers, RVM checks for, and uses, what is the most current
> patchlevel in its $rvm_path/config/db for that Major.Minor version.

**Comment by Stratus3D** [comment]
I'm looking at:

https://rvm.io/workflow/projects
https://gist.github.com/fnichol/1912050

And I do not see anything about `.ruby-version` supporting versions only specifying major and minor version numbers.

**Comment by azhi** [comment]
It isn't explicitly mentioned in the docs, but gist you provided had some discussion on the matter.

https://gist.github.com/fnichol/1912050#gistcomment-628554 (this and following 5-6 comments), https://gist.github.com/fnichol/1912050#gistcomment-804032, https://gist.github.com/fnichol/1912050#gistcomment-1315733.

TL;DR is chruby supports fuzzy matching - any substring in .ruby-version will match rubies containt it, rvm supports specifying `major.minor` - latest installed `major.minor.patch` will be picked, while rbenv is strongly against the idea and supports only full version.

A little demonstration to confirm rvm supports this:

```bash
$ mkdir -p /tmp/ruby-version-test
$ cd /tmp/ruby-version-test
$ rvm current
ruby-2.5.0
$ echo "2.6" > .ruby-version
$ cd ./
$ rvm current
ruby-2.6.1
```

I think note in this PR would help rvm and chruby users when migrating.

**Comment by Stratus3D** [comment]
My point is I'd rather not document this in the readme since it's not something that is explicit in the `.ruby-version` document. That document is the most canonical document on the `.ruby-version` format, even if it is a bit vague. I'd rather only document things it specifies that we do not support. Fuzzy matching isn't something we support, and I'm not sure we will ever support fuzzy matching in the `.ruby-version` file, so I don't want users to get the wrong impression.

It may be helpful to let people coming from rvm know that they will they may have to change the format of their version strings. If you want to reword it so it's clear that the note only pertains to users coming from rvm I think that would be better.

**Comment by azhi** [comment]
Fair enough, change the wording to mention migrating from rvm/chruby.
Does it look better?

**Comment by Stratus3D** [comment]
Thanks! Looks great!
